code/modules/mob/update_icons.dm

```diff
@@ -68,4 +68,7 @@
 	return
 
 /mob/proc/update_inv_ears()
-	return
\ No newline at end of file
+	return
+
+/mob/proc/update_inv_splints()
+	return

```

Comment:
I'd question why this (and the others) are on mob level when only living/carbon can utilize it.